nhl_api_py/core/models.py

```diff
@@ -125,6 +125,84 @@ def from_dict(cls, data: dict):
         return cls(**final_data)
 
 
+@dataclass
+class GeneralGame(Model):
```

Comment:
Lets just call this Game
```suggestion
class Game(Model):
```